fem/integrator.hpp

```diff
@@ -113,6 +113,43 @@ class Integrator
       const ElementTransformation& trans) const
    { return NULL; }
 };
+
+/** @brief This struct contains information about basis functions
+    used in patchwise integration. */
```

Comment:
`patch-wise integration on NURBS meshes` in case someone is unaware that this is for NURBS.